hooks/hookstage/bidderrequest.go

```diff
@@ -25,6 +24,6 @@ type BidderRequest interface {
 // distilled for the particular bidder.
 // Hooks are allowed to modify openrtb2.BidRequest using mutations.
 type BidderRequestPayload struct {
-	BidRequest *openrtb2.BidRequest
-	Bidder     string
+	RequestWrapper *openrtb_ext.RequestWrapper
```

Comment:
IMHO the RequestWrapper is an abstraction on top of the bid request, but it still represents a bid request. I recommend leaving this variable name (and others) as "BidRequest" of type RequestWrapper.

Reply:
Renamed. Also renamed `RequestWrapper` to `BidRequest` in `ProcessedAuctionRequestPayload`